spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/FeedForwardNetwork.scala

```diff
@@ -36,12 +36,13 @@ class FeedForwardNetwork[T: ClassTag](val hiddenSize: Int, val filterSize: Int,
   override def buildModel(): Module[T] = {
     val input = Input()
     val filterLayer = TransformerOperation.dense(
-      hiddenSize, filterSize, bias = true, activation = ReLU[T]()).inputs(input)
+      hiddenSize, filterSize, bias = true, activation = ReLU[T](),
+      name = s"${this.getName()}_filter_layer").inputs(input)
```

Comment:
why add module name ?

Reply:
We have to copy tf weights to ours when doing unit tests.
If no name, it is hard to match correctly.